### DISCUSSION TITLE
新版本丢配置 希望保留老版本安装入口

### DISCUSSION THREAD

**Discussion by ts8zs** [the1812/Bilibili-Evolved#3431] [功能建议 / Ideas]
win10 Edge 暴力猴 
在一个页面安装插件后刷新插件丢失

**Comment by Geeyun-JY3** [comment]
确定不是 #3331 的问题？

**Reply by ts8zs** [reply]
如果是那个问题插件应该搜不出来吧?

**Comment by the1812** [comment]
老版本: https://github.com/the1812/Bilibili-Evolved/raw/master/bilibili-evolved.offline.user.js
